### PR TITLE
[Website] Ft upgrade packages

### PR DESCRIPTION
**Pull request by FaithDaka**
#### Summary of Changes (What does this PR do?)

- Specifies node engines required to run the app (node 14+)
- Upgrades node packages

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).

#### How should this be manually tested?

- Navigate through the app in deploy preview to see if any page breaks or errors returned

#### What are the relevant tickets?

- [WEB-363]

[WEB-363]: https://airqoteam.atlassian.net/browse/WEB-363?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Baalmart — COMMENTED** [review]
Thanks @FaithDaka , this is good stuff!

Should we be expecting a deploy preview for this PR? I can't seem to find it.

Thanks!

**Review by 123MwanjeMike — APPROVED** [review]
Thank you @FaithDaka 
The JS package updates LGTM
image

**Comment by 123MwanjeMike** [comment]
> Should we be expecting a deploy preview for this PR? I can't seem to find it.

Yes @Baalmart 
That was because @FaithDaka's changes needed a corresponding Dockerfile update for the deploy preview to work.
Note that these have been implemented

File: website/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Building frontend
-FROM node:12-slim as frontend
+FROM node:18-slim as frontend
```
Comment (123MwanjeMike) [inline website/Dockerfile:2]: Upgrading node base image to use

**Review by Baalmart — APPROVED** [review]

**Pull request merged**